regression/apps/mongoose/Makefile

```diff
@@ -1,47 +1,39 @@
 # SPDX-License-Identifier: MPL-2.0
 
-MAIN_MAKEFILE := $(firstword $(MAKEFILE_LIST))
-INCLUDE_MAKEFILE := $(lastword $(MAKEFILE_LIST))
-CUR_DIR := $(shell dirname $(realpath $(MAIN_MAKEFILE)))
+CUR_DIR := $(shell dirname $(realpath $(firstword $(MAKEFILE_LIST))))
 BUILD_DIR := $(CUR_DIR)/../../build/initramfs/regression/network
+MONGOOSE_DIR := $(CUR_DIR)/../../build/mongoose
+MONGOOSE_C := $(MONGOOSE_DIR)/mongoose.c
+MONGOOSE_H := $(MONGOOSE_DIR)/mongoose.h
+MONGOOSE_FILES := $(MONGOOSE_C) $(MONGOOSE_H)
+MONGOOSE_O := $(MONGOOSE_DIR)/mongoose.o
+SRC_SERVER := http_server.c
+SRC_CLIENT := http_client.c
 TARGET_SERVER := $(BUILD_DIR)/http_server
 TARGET_CLIENT := $(BUILD_DIR)/http_client
+BINS := $(TARGET_SERVER) $(TARGET_CLIENT)
```

Comment:
Things should be named consistently. Now that you have `MONGOOSE_C`, don't you think `http_server.c` should be named `SERVER_C` instead of `SRC_SERVER`? All the generated binaries are covered by a variable named `BINS`, but its elements are named `TARGET_XXX`, instead of `XXX_BIN`?

So I suggest making the following changes:

```Makefile
SERVER_C := http_server.c
SERVER_BIN := $(BUILD_DIR)/http_server
CLIENT_C := http_client.c
CLIENT_BIN := $(BUILD_DIR)/http_client
BINS := $(SERVER_BIN) $(CLIENT_BIN)
```

Reply:
It looks better this way!